src/views/DocumentationTopic.vue

```diff
@@ -328,60 +247,12 @@ export default {
     applyObjcOverrides() {
       this.topicDataObjc = apply(clone(this.topicData), this.objcOverrides);
     },
-    handleCodeColorsChange(codeColors) {
-      CodeThemeStore.updateCodeColors(codeColors);
-    },
-    handleToggleSidenav(breakpoint) {
-      if (breakpoint === BreakpointName.large) {
-        this.toggleLargeSidenav();
-      } else {
-        this.toggleMobileSidenav();
-      }
-    },
-    openQuickNavigationModal() {
-      if (this.sidenavVisibleOnMobile) return;
-      this.showQuickNavigationModal = true;
-    },
-    toggleLargeSidenav(value = !this.sidenavHiddenOnLarge) {
-      this.sidenavHiddenOnLarge = value;
-      storage.set(NAVIGATOR_HIDDEN_ON_LARGE_KEY, value);
-    },
-    toggleMobileSidenav(value = !this.sidenavVisibleOnMobile) {
-      this.sidenavVisibleOnMobile = value;
-    },
-    onQuickNavigationKeydown(event) {
-      // Open the modal only on `/` or `cmd+shift+o` key event
-      if (event.key !== '/' && !(event.key === 'o' && event.shiftKey && event.metaKey)) return;
-      // Prevent modal from opening when the navigator is disabled
-      if (!this.enableNavigator) return;
-      // Prevent modal from opening if the event source element is an input
-      if (event.target.tagName.toLowerCase() === 'input') return;
-      this.openQuickNavigationModal();
-      event.preventDefault();
-    },
   },
   mounted() {
     this.$bridge.on('contentUpdate', this.handleContentUpdateFromBridge);
-    this.$bridge.on('codeColors', this.handleCodeColorsChange);
-    this.$bridge.send({ type: 'requestCodeColors' });
-    if (this.enableQuickNavigation) window.addEventListener('keydown', this.onQuickNavigationKeydown);
-  },
-  provide() {
-    return {
-      store: this.store,
-    };
```

Comment:
I think that since it's provided in the [DocumentationLayout](https://github.com/apple/swift-docc-render/pull/792/files#diff-574b009e870640efdf5cc987be4d93c7dd31d603b04f7ac566abc6e99076f732R149), it doesn't need to be provided here too, right?

Reply:
Yes, I deleted the store from `DocumentationLayout` later on https://github.com/apple/swift-docc-render/pull/792/commits/812f82f54b0b03aea853fb843e1dcb9ebfdc5b27

This comment was outdated, sorry 🙈